### ISSUE TITLE
Hello Dolly Plugin: When active it hides the Screen Options and Help tabs in Installed Plugins section

### ISSUE THREAD

**Issue by zdenys** [Automattic/wp-calypso#60361]
### Quick summary

When the plugin is active, it overlaps/hides the Screen Options and Help tabs in Plugins > Installed Plugins section. That prevents users from [switching back to WordPress.com view](https://wordpress.com/support/dashboard/#switch-to-word-press-com-view).

![image](https://user-images.githubusercontent.com/39308239/150640724-34e2aeff-ac9b-40ab-8a30-009aa5516d3c.png)

I think it should be fixed on our end and not in the plugin itself because we created the Nav Unification project and the related Screen Options to switch between two views. 

The related issue is also here: https://github.com/Automattic/wp-calypso/issues/50508

### Steps to reproduce

1. Install and activate Hello Dolly https://wordpress.org/plugins/hello-dolly/
2. Navigate to Plugins > Installed Plugins

### What you expected to happen

Continue to see the Screen Options and Help tabs at the top right

### What actually happened

The Screen Options and Help tabs are not shown and the horizontal stripe with quotes shows in that same position and it's created by Hello Dolly plugin.

![image](https://user-images.githubusercontent.com/39308239/150640724-34e2aeff-ac9b-40ab-8a30-009aa5516d3c.png)

### Simple, Atomic or both?

Atomic

### Theme-specific issue?

No

### Browser, operating system and other notes

macOS Monterrey 12.1 with Chrome 97

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

Turn off Hello Dolly.

**Comment by Greatdane** [comment]
Confirmed. Maybe we should ask Matt to update the plugin too 😅

**Comment by zdenys** [comment]
@Greatdane I think it's the technical debt of the Nav Unification project actually on WordPress.com only. Hello Dolly plugin works fine on WordPress.org. So I don't think the plugin needs any changes.

**Comment by wojtekn** [comment]
I've checked this issue, reproduced it, and found a possible fix.

The styles for Hello Dolly UI look good in the plugin itself. The styling for the navigation bar also looks good on both WP.org and WP.com.

The problem looks like introduced in the Crowdsignal Forms plugin. It overwrites styles for the Hello Dolly box using the `.wp-admin #dolly` selector, and that styling doesn't take into account the tabs placed under the navigation bar:

https://github.com/Automattic/crowdsignal-forms/blob/cd136e5338b98b10e23ee1b9d8939f5b8165adda/includes/admin/admin-styles.css#L85

The problem pops out on Atomic sites, but it would also pop out on the WP.org site with both Crowdsignal Forms and Hello Dolly plugins installed.

The fix I played with is about removing the whole custom CSS for Hello Holly from the Crowdsignal Forms plugin source. When I deleted that code, Hello Dolly's look and feel were consistent and floated correctly into the navigation bar tabs.

I'm unassigning this issue from me for now to let me focus on other issues.
